pytest.sh

```diff
@@ -18,7 +18,7 @@ pip3 install -r requirements-test.txt
 find . -name "*.pyc" -type f -delete
 find . -name "__pycache__" -type d -delete
 { # try
-    pytest tests/functional tests/ui_tests -s -v -n auto --maxfail 30 \
+    pytest tests/functional/test_adcm_upgrade.py -s -v -n auto --maxfail 30 \
```

Comment:
If you wanna start your life from scratch, then just drop all previous tests.

Reply:
Lol. Fixed